src/MizarWidgets/SamplingWithFrameTrackInputWidgetTest.cpp

```diff
@@ -67,8 +67,8 @@ class MockPairedData {
               (const));
 };
 
```

Comment:
Unrelated: In my opinion this anonymous namespace should still be inside `namespace orbit_mizar_widgets {`.

Reply:
SG. I am going to tackle this right in a follow-up